spec/features/attach_agreement_to_user_spec.rb

```diff
@@ -19,7 +19,7 @@
     click_link('Update/Verify Profile')
     expect(page).to have_content('Edit Staff Account')
 
-    page.attach_file('user_agreement_attributes_attachment', 'public/docs/spay-neuter-agreement.pdf')
+    page.attach_file('user_agreement_attributes_attachment', Rails.root + 'public/docs/spay-neuter-agreement.pdf')
```

Comment:
Existing, but I think the preference is to use `Rails.root.join("public", "docs", "spay-neuter-agreement.pdf") for example to get the appropriate OS-specific slash direction.